direct/src/distributed/CRDataCache.py

```diff
@@ -81,11 +81,11 @@ def flush(self):
     dc._startMemLeakCheck()
 
     cd = CachedDOData()
-    cd.foo = 34
+    cd.foo = 34  # type: ignore[attr-defined]
```

Comment:
Is there no way to mark CachedDOData as a class that stores arbitrary attributes?  I assume this will fail with mypyc.

Reply:
Ah, I hadn't realized that was the entire point of `CachedDOData`. Yes, that should be possible.